common/components/PrintableWallet/index.jsx

```diff
@@ -0,0 +1,246 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import PropTypes from 'prop-types';
+import translate from 'translations';
+import QRCode from 'qrcode';
+import { toDataUrl as makeIdenticon } from 'ethereum-blockies';
+
+import ethLogo from 'assets/images/logo-ethereum-1.png';
+import sidebarImg from 'assets/images/print-sidebar.png';
+import notesBg from 'assets/images/notes-bg.png';
+
+const walletWidth = 680;
+const walletHeight = 280;
+
+export default class PrintableWallet extends Component {
+  static propTypes = {
+    privateKey: PropTypes.string,
+    address: PropTypes.string
+  };
+
+  state = {
+    qrCodePkey: null,
+    qrCodeAddress: null
+  };
+
+  componentDidMount() {
+    // Start generating QR Codes as soon as component is ready
+    const opts = {
+      color: {
+        dark: '#000',
+        light: '#fff'
```

Comment:
There are two calls to `toDataURL` here, I didn't want to repeat the object twice.